test/bulk/.env

```diff
@@ -1,3 +1,4 @@
 SKU=RE-VRC-23-1488
 CODE=C0F-BEF
-DIVISION=1
\ No newline at end of file
+DIVISION=1
+VITE_REFEREE_FYI_BUILD_MODE=WC
```

Comment:
Sorry I should have been more clear. The root directory of the project, you should make a .env file (it's gitignored) to include that setting